packages/shared/src/components/search/SearchPanel/SearchPanel.tsx

```diff
@@ -191,12 +191,15 @@ export const SearchPanel = ({ className }: SearchPanelProps): ReactElement => {
           }}
         >
           {showDropdown && (
+            
             <div
               className={classNames(
-                'absolute w-full items-center rounded-b-16 border-0 border-border-subtlest-tertiary bg-background-default px-3 py-2 laptop:h-auto laptop:border-x laptop:border-b laptop:bg-background-subtle laptop:shadow-2',
+                'absolute w-full items-center rounded-b-16 border-0 border-border-subtlest-tertiary bg-background-default px-3 py-2 laptop:h-auto laptop:border-x laptop:border-b laptop:bg-background-subtle laptop:shadow-2'
```

Comment:
Through tailwind, we should be able to apply the styles.
```suggestion
                'absolute w-full items-center rounded-b-16 border-0 border-border-subtlest-tertiary bg-background-default px-3 py-2 laptop:h-auto laptop:border-x laptop:border-b laptop:bg-background-subtle laptop:shadow-2 overflow-y-auto max-h-[30rem]'
```